modules/developer-cli-odo-creating-services-from-yaml-files.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc
+
+[id="creating-services-from-yaml-files_{context}"]
+
+= Creating services from YAML files
+
+If the YAML definition of the service or Custom Resource (CR) has invalid or placeholder data , you can use the `--dry-run` flag to get the YAML definition, specify the correct values and start the service using the corrected YAML definition.
```

Comment:
s/data ,/data, 
s/correct values and/correct values, and
s/service (Custom Resource or CR)/service or CR